apps/web/languages/en.json

```diff
@@ -25,6 +25,24 @@
         "popover": {
           "title": "Settings"
         }
+      },
+      "modal": {
+        "titleRegister": "Sign up",
+        "subTitleRegister": "By registering, will be able to store your data.",
+        "formSignUp": {
+          "placeholderName": "Name",
+          "placeholderEmail": "Email",
+          "placeholderPassword": "Password",
+          "createButton": "Create account"
+        },
+
+        "titleLogin": "Sign in",
+        "subTitleLogin": "Keep logging into your account.",
+        "formSignIn": {
+          "placeholderEmail": "Email",
+          "placeholderPassword": "Password",
+          "createButton": "Continue"
+        }
```

Comment:
@MFelipeSilva we dont need a modal object

Reply:
ok, adjusted